style.css

```diff
@@ -0,0 +1,169 @@
+html,
+body {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    font-size: 100%;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    background: linear-gradient(180deg, rgba(255, 218, 185, 1) 0%, rgba(248, 173, 157, 1) 100%);
+    transition: background-color 2s ease;
+}
+
+header {
+    height: 33.33%;
+    position: relative;
+}
+
+main {
+    height: 66.66%;
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+}
+
+h1 {
+    text-align: center;
+    position: absolute;
+    top: 20%;
+    right: 50%;
+    transform: translateX(50%);
+    font-size: 5em;
+    border-bottom: 5px solid #f08080;
+    border-image: linear-gradient(90deg, rgb(252, 129, 129) 0%, rgb(252, 205, 179) 100%);
+    border-image-slice: 1;
+}
+
+.secret {
+    height: 5rem;
+    width: 5rem;
+    text-align: center;
+    font-size: 4em;
+    position: absolute;
+    top: 90%;
+    right: 50%;
+    transform: translateX(50%);
+    background-color: #f8ad9d;
+    border-radius: 20px;
+}
+
+.range {
+    position: absolute;
+    top: 2rem;
+    left: 85%;
+    text-align: center;
+    font-size: larger;
+}
+
+.replayBtn:hover {
+    background-color: #f8ad9d;
+}
+
+.replayBtn:active {
+    background-color: #f4978e;
+}
+
+.replayBtn {
+    position: absolute;
+    top: 3rem;
+    right: 85%;
+    width: 6em;
+    height: 2em;
+    font-size: 1.5em;
+    font-family: 'Roboto', sans-serif;
+    background-color: #fbc4ab;
+    border: none;
+    border-radius: 10px;
+    cursor: pointer;
+}
+
+.left {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 2em;
+}
+
+.userInput:focus {
+    outline: none;
+}
+
+.userInput {
+    font-family: 'Roboto', sans-serif;
+    background: none;
+    width: 15rem;
+    height: 8rem;
+    font-size: 5rem;
+    border: 12px solid #f4978e;
+    border-radius: 10px;
+    text-align: center;
+
+
+}
+
+.checkBtn:hover {
+    background-color: #f8ad9d;
+}
+
+.checkBtn:active {
+    background-color: #f4978e;
+}
+
+.checkBtn {
+    background-color: #fbc4ab;
+    font-family: 'Roboto', sans-serif;
+    font-size: 2em;
+    font-weight: 700;
+    border: none;
+    border-radius: 10px;
+    text-align: center;
+    padding: 1rem 2rem;
+    cursor: pointer;
+}
```

Comment:
+ You dit not take into account my previous comment on how to factorize css class, you should,
+ I don't think camelCase is recommended for css, probably something like `check-button`
+ the `font-family: 'Roboto', sans-serif;` is already inherited from the `body` css class why bothering repeat it here ?